configs/small-scale-node-density-cni.yml

```diff
@@ -0,0 +1,58 @@
+tests :
+  - name : aws-small-scale-node-density-cni
+    index: ospst-perf-scale-ci-*
+    benchmarkIndex: ospst-ripsaw-kube-burner*
+    metadata:
+      platform: AWS
+      masterNodesType: m6a.xlarge
+      masterNodesCount: 3
+      workerNodesType: m6a.xlarge
+      workerNodesCount: 6
+      infraNodesCount: 3
+      benchmark.keyword: node-density-cni
+      ocpVersion: {{ version }}
```

Comment:
could we maybe put in the title template here so that a user doesn't take this file and just try to run as this version is not filled in

Reply:
Yes, that makes sense, will add a comment mentioning it a template.